README.md

```
@@ -0,0 +1,57 @@
+| Component | Technology |
+|-----------|-----------|
+| Backend API | .NET 10, ASP.NET Core, Native AOT |
+| Database | Azure Cosmos DB (Serverless) |
+| iOS App | Swift, SwiftUI, SwiftData |
+| Infrastructure | Pulumi (C# / .NET 10), Azure Container Apps |
+| CI/CD | GitHub Actions |
+| Testing | TUnit (.NET), XCTest (iOS) |
+
+## Repository Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)